src/libraries/Common/tests/System/DirectoryServices/LDAP.Configuration.xml

```diff
@@ -6,11 +6,11 @@ To ship, we should test on both an Active Directory LDAP server, and at least on
 
 When testing with later of versions of LDAP, the ldapsearch commands below may need to use
 
-    -H ldap://localhost:<PORT>
```

Comment:
This actually fixes a recent regression since the manually-run tests will try to read this as an XML file, and the use of `<` broke that.

Reply:
Per our discussion - it looks like those weren't running in CI due to the `LdapConfigurationExists` check causing them to be skipped.  If they should run we can file an issue to track fixing that.

Reply:
I created https://github.com/dotnet/runtime/issues/112108 to track that and a couple other test areas.